public/app/percona/shared/helpers/getValuesFromQueryParams.ts

```diff
@@ -0,0 +1,31 @@
+import { UrlQueryMap, UrlQueryValue } from '@grafana/data';
+
+interface QueryParamTransform {
+  key: string;
+  transform?: (param: UrlQueryValue) => any;
+}
+
+const defaultTransform = (params: UrlQueryValue): string[] => {
+  if (params && params !== undefined && params !== null) {
+    return typeof params === 'object' ? [...params.map((p) => String(p))] : [String(params)];
```

Comment:
Why the brackets with spread operator? map returns new array.

Reply:
Right!